src/test/unit_tests/test_nc_cli.js

```diff
@@ -1055,6 +1055,22 @@ mocha.describe('manage_nsfs cli', function() {
                 assert_error(err, ManageCLIError.InvalidArgument);
             }
         });
+
+        mocha.it('cli add whitelist config file doesnt exist', async function() {
+            //delete existing config file
+            const config_file_path = config_fs.get_config_json_path();
+            await fs_utils.file_delete(config_file_path);
+
+            const new_config_options = {};
+            const ips = ['127.0.0.1']; // IPV4 format
+            const res = await exec_manage_cli(type, '', { config_root, ips: JSON.stringify(ips) });
+            new_config_options.S3_SERVER_IP_WHITELIST = ips;
+            const config_data = await config_fs.get_config_json();
+            console.log(config_data);
+            await assert_response('', type, res, ips);
```

Comment:
can you move this assert right after the `exec_manage_cli()`, there is no point in fetching config_json(config_fs.get_config_json()) if this assert fails.